app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,33 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+/* universal */
+body {
+  padding-top: 60px;
+}
+section {
+  overflow: auto;
+}
+textarea {
+  resize: vertical;
+}
+.center {
+  text-align: center;
+}
+.center h1 {
+  margin-bottom: 10px;
+}
+/* header */
```

```suggestion

/* header */
```
trước line comment thì có ine trống nhé bé bì

---

ready